src/test/kotlin/calculator/ExpressionTest.kt

```diff
@@ -0,0 +1,59 @@
+package calculator
+
+import org.assertj.core.api.Assertions.assertThat
+import org.assertj.core.api.Assertions.assertThatIllegalArgumentException
+import org.junit.jupiter.api.assertThrows
+import org.junit.jupiter.params.ParameterizedTest
+import org.junit.jupiter.params.provider.Arguments
+import org.junit.jupiter.params.provider.MethodSource
+import java.util.stream.Stream
+
+internal class ExpressionTest {
+
+    @ParameterizedTest
+    @MethodSource("generate")
+    fun `계산하기`(operands: MutableList<Int>, operators: MutableList<String>, actual: Int) {
+
+        val expected = Expression.calculate(operands, operators)
+
+        assertThat(expected).isEqualTo(actual)
+    }
+
+    @ParameterizedTest
+    @MethodSource("failGenerate")
+    fun `연산자가 아닌 다른 문자열`(operands: MutableList<Int>, operators: MutableList<String>, actual: Int) {
+
+        assertThatIllegalArgumentException()
+            .isThrownBy { Expression.calculate(operands, operators) }
```

Comment:
```suggestion
        assertThrows<IllegalArgumentException> { Expression.calculate(operands, operators) }
```
`assertThatIllegalArgumentException()` 를 사용하면 `Assertions.java` 를 사용하게 됩니다. `Assertions`를 `Kotlin`에 맞게 구현 된 `assertThrows<>`를 사용해보시는건 어떨까요? 🤔

Reply:
네~ 다양하게 해보려다 이렇게 되었네용~